### PR TITLE
Change mingw-build to use download URLs

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

File: mingw-build.cmd [hunk 1]
```diff
@@ -7,17 +7,17 @@ REM the PATH environment needs to include 7z.exe, mingw32-make.exe and a working
 REM installation.
 set "PATH=C:\Program Files\7-Zip;C:\TDM-GCC-32\bin;%PATH%"
 
-set "CMAKE_URL=file://C:/Users/joeub/Downloads/cmake-3.16.0-win32-x86.zip"
-set "CMAKE_FILENAME=cmake-3.16.0-win32-x86.zip"
-set "CMAKE_DIRNAME=cmake-3.16.0-win32-x86"
-set "BZIP2_URL=file://C:/Users/joeub/Downloads/bzip2-1.0.8.tar.gz"
+set "CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v3.17.3/cmake-3.17.3-win32-x86.zip"
+set "CMAKE_FILENAME=cmake-3.17.3-win32-x86.zip"
+set "CMAKE_DIRNAME=cmake-3.17.3-win32-x86"
```
Comment (jmeubank) [inline mingw-build.cmd:12]: Version changes will require testing. Everything else looks good to me.

Comment (mimi89999) [reply mingw-build.cmd:12]: It builds fine. What else should be tested? I can downgrade to 3.16.0 if you want.

**Comment by jmeubank** [comment]
Thanks for taking the time to contribute the first pull request @mimi89999 ! I'm very happy to receive it and in general this is a good enhancement. 😃 Testing successful, will merge.

**Comment by mimi89999** [comment]
Thanks. I had to build xarc to build tdm installer (that has a similar PR open) so I 'fixed' the build and contributed it.

**Pull request merged**